charts/redpanda/Chart.yaml

```diff
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 5.7.37
+version: 5.7.38
```

Comment:
A linter will complain that you haven't updated the README.md. I don't think I've made a task for that yet but you can find the command to run under the `ci:lint` task definition.